### PR TITLE
sysbench: add a 'insert ignore' bench test with unique indices

### PR DESCRIPTION
**Pull request by tiancaiamao**
Benchmark for https://github.com/pingcap/tidb/pull/12872

PTAL @jackysp

### CODE REVIEW

File: sysbench/oltp_insert_ignore.lua [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                           table_num, table_num))
+end
+
+function prepare_statements()
+end
+
+function event()
+  local table_name = "sbtest" .. sysbench.rand.uniform(1, sysbench.opt.tables)
+  local query = string.format("INSERT IGNORE INTO %s (id, k, c, pad) VALUES ", table_name)
+  for i = 1, 5 do
```
Comment (jackysp) [inline sysbench/oltp_insert_ignore.lua]: should we use a larger number, I think 5 is a bit small.

Comment (tiancaiamao) [reply sysbench/oltp_insert_ignore.lua]: This is not a prepared statement, if we use a lot of VALUES, the performance would be affected by the parser (the dominant time cost) and then the test can't reflect the truth.

5 is enough to cover insert ignore code and prevent the performance regression if a request visit TiKV multiple times.

**Comment by tiancaiamao** [comment]
PTAL @mahjonp

File: sysbench/oltp_insert_ignore.lua (hunk 1 above)
Comment (jackysp) [reply sysbench/oltp_insert_ignore.lua]: It is OK. Below 100 values, the effects of parser should be negligible. Also, you could also use the prepare statement.

Comment (tiancaiamao) [reply sysbench/oltp_insert_ignore.lua]: > Also, you could also use the prepare statement.

emmmm... I'd like to but I'm not so familiar with Lua this bench framework... so just get things done

Comment (mahjonp) [reply sysbench/oltp_insert_ignore.lua]: How about adding a parameter: Eg, https://github.com/pingcap/productivity-tools/blob/master/benchmark/batch_insert/batch_insert.lua#L65

**Pull request merged**